src/ttf2pt1/ttf2pt1.c

```diff
@@ -656,7 +656,7 @@ unicode_init_user(
 		if( !enabled )
 			continue; /* skip to the next plane */
 
-		if( sscanf(buffer, "at %i", &curpos) == 1 ) {
```

Comment:
This may look a bit odd, casting the pointer to an `int*`. It's done because I'm not sure if it's safe to convert `%i` to, say, `%u`.

It would be safe to change from `%d` to `%u`, because those both operate on strings of decimal digits. However, `%i` can accepts decimal hexadecimal, and octal digits. I don't know if the actual input has hexadecimal or octal digits, so it seemed safer to simply cast the pointer, especially since the code has worked fine for all these years.